gammapy/utils/parallel.py

```diff
@@ -9,14 +9,14 @@
 
 
 class ParallelBackendEnum(Enum):
-    """Enum for parallel backend"""
+    """Enum for parallel backend."""
```

Comment:
Here `Enum` is a keyword for `Enumeration`, so do we want to put the full name or keep the keyword ?

Reply:
As the docstring is so short I thought keeping it as the keyword is best